### PR TITLE
added android splash screen on native way

### PR DESCRIPTION
**Pull request by matze1128**
Added nice and clean splash screen.

### CODE REVIEW

File: fluttr_app/explorer_app/android/app/src/main/res/drawable/launch_background.xml [hunk 1]
```diff
@@ -1,12 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Modify this file to customize your launch splash screen -->
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:drawable="@android:color/white" />
+    <item android:drawable="@color/grey" />
```
Comment (christian-heusel) [inline fluttr_app/explorer_app/android/app/src/main/res/drawable/launch_background.xml:3]: Why did you choose grey here? Imo white would have looked better 😊

File: fluttr_app/explorer_app/api [hunk 2]
```diff
@@ -1 +1 @@
-../../api
\ No newline at end of file
+../../api-
```
Comment (christian-heusel) [inline fluttr_app/explorer_app/api]: Please dont (ever) do this again xD

**Review by christian-heusel — CHANGES_REQUESTED** [review]

File: fluttr_app/explorer_app/android/app/src/main/res/drawable/launch_background.xml (hunk 1 above)
Comment (matze1128) [reply fluttr_app/explorer_app/android/app/src/main/res/drawable/launch_background.xml:3]: I think a grey color theme fits to the blue of our logo.

File: fluttr_app/explorer_app/api (hunk 2 above)
Comment (matze1128) [reply fluttr_app/explorer_app/api]: ups

**Review by christian-heusel — APPROVED** [review]

**Pull request merged**